kuma/wiki/forms.py

```diff
@@ -466,15 +467,25 @@ def akismet_parameters(self):
         for a model instance in
         ``RevisionAkismetSubmissionAdminForm.akismet_parameters`` method!
         """
-        language = self.cleaned_data.get('locale',
-                                         settings.WIKI_DEFAULT_LANGUAGE)
+        if self.instance and self.instance.pk and self.instance.document:
+            language = self.instance.document.locale
+        else:
+            language = settings.WIKI_DEFAULT_LANGUAGE
+
```

Comment:
I'd rather have an "akismet_locale" method that is customized for each form class that does the right thing for that form's data. The priority is getting the language right for our translators, which is currently broken, so pick a solution based on what gets that fixed early next week, not what gets an A in the code design class.  I'll be revisiting this code several times in the next few weeks, and hopefully the refactors will be obvious by then.